auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,414 @@
+
+All these values of MOI Tensor can be calculated as follows:
+
+$$\begin{eqnarray} 
+ I\_{11} = I\_{xx} = \int (y^2 + z^2)dm \\\
+ I\_{22} = I\_{yy} = \int (x^2 + z^2)dm \\\
+ I\_{33} = I\_{zz} = \int (x^2 + y^2)dm \\\
+\end{eqnarray}$$
+
+Here \\(dm\\) is the mass of an infinitesimal unit of the object and \\(x, y, z\\) are the distances of that unit from axes. 
+
+Similarly, the Products of Inertia can be calculated as:
+
+$$\begin{eqnarray}
+ I\_{12}  = I\_{xy} = \int -xydm = I\_{yx} = I\_{21} \\\
+ I\_{13}  = I\_{xz} = \int -xzdm = I\_{zx} = I\_{31} \\\
+ I\_{23}  = I\_{yz} = \int -yzdm = I\_{zy} = I\_{32} \\\
+\end{eqnarray}$$
+
+In this solution, the **infinitesimal element** of the object can be **represented by the each Voxel.** 
+Instead of calculating the mass \\(dm\\) for each voxel, we will calculate the volume \\(dv\\) of each 
+voxel using the voxel size (the simple volume formula of a cube can be used because voxels are cubes). 
+Then mass for each element would be mass density, \\(\rho\\) multiplied by the volume, \\(dv\\). 
+Considering the mass density to be constant and substituting in the above equations we get:
+
+$$\begin{eqnarray}
+I\_{11} = I\_{xx} = \rho\int (y^2 + z^2)dv \\\
+I\_{22} = I\_{yy} = \rho\int (x^2 + z^2)dv \\\
+I\_{33} = I\_{zz} = \rho\int (x^2 + y^2)dv \\\
+I\_{12}  = I\_{xy} = \rho\int -xydv = I\_{yx} = I\_{21} \\\
+I\_{13}  = I\_{xz} = \rho\int -xzdv = I\_{zx} = I\_{31} \\\
+I\_{23}  = I\_{yz} = \rho\int -yzdv = I\_{zy} = I\_{32} \\\
+\end{eqnarray}$$
+
+#### Advantages of the Voxelization Approach
+ * Voxelization is a generalized approach that would work for all kinds 
+ of meshes (convex and non-convex meshes).
+ * It can be made configurable by allowing the user to set an appropriate `voxel_size` 
+ for the process. A smaller `voxel_size` would be comparitively computationally heavy 
+ but would provide closer to real world value for moment of inertia.
+ * Voxelization is more intuitive way of moment of inertia calculations as 
+ compared to other integral methods.
+
+### Integration-based numerical method
+This method utilizes Gauss’s Theorem and Greene’s Theorem of integration. First, we convert volume 
+integrals to surface integrals using Gauss’s Theorem and then surface integrals to line 
+integrals using Greene’s Theorem<sup>[\[5\]](#References)</sup>.
+This method works for triangle meshes which are simple water-tight polyhedrons. 
+Currently, the origin of the mesh being used needs to be set at the geometric center 
+to obtain the correct value.
+
+#### Some Key Points Regarding the Integration-based numerical method
+
+ * Water-tight triangle meshes are required for the Mesh Inertia Calculator.
+
+ * The inertia in this method is calculated about the mesh origin. Since the link inertia value needs to be about the Center of Mass, the mesh origin needs to be set at the Center of Mass (Centroid) of the mesh.
```

Comment:
This is discussing something implemented in gz-sim, so perhaps we should say there's a reference implementation in gz-sim and point to the file before listing these key points. Alternatively, we could remove this altogether since it'll be documented in gz-sim.

Reply:
I have added a link to the reference implementation in [52b582e](https://github.com/gazebosim/sdf_tutorials/pull/92/commits/52b582ee7c155e32a76fe28b278d6af3276b5967)